roles/hammerdb/tasks/main.yaml

```diff
@@ -0,0 +1,64 @@
+---
+
+- name: Show template variables
+  debug:
+    msg: "What do we do, init {{hammerdb.db_init}} or benchmark {{hammerdb.db_benchmark}}"
+
+- name: template db creation script
+  k8s: 
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata: 
+        name: '{{ meta.name }}-hammerdb-config'
```

Comment:
following pr #191 any resources created should make use of the `trunc_uuid` var appended to name and label.

Reply:
fixed